doc/source/development/code_style.rst

```diff
@@ -128,6 +128,11 @@ For example:
     value = str
     f"Unknown received type, got: '{type(value).__name__}'"
 
+PEP8 / black
+------------
+pandas follows the PEP8 standard and uses Black and Flake8 to ensure a
+consistent code format throughout the project. For details see the
+contributing guide https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#python-pep8-black.
```

Comment:
This should be using a sphinx reference, so it's automativally rendered as a link, and not the url as text.